metricbeat/mb/metricset/http.go

```diff
@@ -0,0 +1,71 @@
+package metricset
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/elastic/beats/metricbeat/mb"
+)
+
+type Http struct {
+	mb.BaseMetricSet
+	client *http.Client // HTTP client that is reused across requests.
+}
+
+// NewHttpMetricset creates new instance of HttpMetriSset
```

Comment:
Yeah, in the first version it was call HttpMetricset because it was not in the metricset package. 

You are right based on our standard it should be called `HTTP`, but it will look wiered :-( I would prefer to leave it. @urso Thoughts.